e2e-tests/playwright/support/page-objects/page-obj.ts

```diff
@@ -1,3 +1,5 @@
+import { Page, Locator } from "@playwright/test";
+import { SemanticSelectors } from "../selectors/semantic-selectors";
```

Comment:
I don't see many new methods actually in use. Do you expect to use them in future refactorings? Are they all tested and reliable?